src/object/cli_obj.c

```diff
@@ -620,8 +620,7 @@ obj_dkey2grpidx(struct dc_object *obj, uint64_t hash, unsigned int map_ver)
 
 	D_ASSERT(obj->cob_shards_nr >= grp_size);
 
-	/* XXX, consistent hash? */
-	grp_idx = hash % (obj->cob_shards_nr / grp_size);
+	grp_idx = d_hash_jump(hash, obj->cob_shards_nr / grp_size);
```

Comment:
Will JCH make sure the integer dkey evenly distributed over shards? probably fine.

Reply:
Yes, it does, it's better than (murmur_hash(integer) % buckets)

Reply:
Actually I mean compared with offset % buckets (original integer dkey with DFS), which is even completely.
Interesting we do similar for lustre, why there is no complain there.